docs/README.md

```diff
@@ -1,66 +1,9 @@
-# Cosmos SDK Documentation
+# Welcome to the Cosmos Docs!
 
-NOTE: This documentation is a work-in-progress!
+![cosmonaut reading the cosmos docs in space](./graphics/cosmos-docs.jpg)
 
-- [Overview](overview) 
```

Comment:
sorry about this. this is a great TOC - do we want to maintain two?

Reply:
i think as long as the vuepress config is updated to roughly match this outline (i.e., nothing important was lost)